### PR TITLE
Update build scripts to comply with bes-bld-isolate

### PR DESCRIPTION
**Pull request by anmol372**
- Prevent concurrent builds
- Wipe workspace after build and Update path to code sign binary

### CODE REVIEW

File: rpm/sign-rpm.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -x
 
 # Ensure the script is called with the correct number of arguments
 if [ "$#" -ne 4 ]; then
-    #echo "Usage: <rpm_path> <base64_vault_role> <base64_vault_sec> <build_user1> <build_user2>"
+    #echo "Usage: <rpm_path> <build_user1> <build_user2> <release-build>"
```
Comment (tbachman) [inline rpm/sign-rpm.sh]: This is commented out. Do we need this line changed?

Comment (anmol372) [reply rpm/sign-rpm.sh]: The earlier comment was incorrect. The change is not necessary as it is commented out.

Comment (tbachman) [reply rpm/sign-rpm.sh]: Is this meant to be helpful for users? If so, should we just get rid of the "echo" and have a comment there saying its usage? If it's not meant to be helpful for users, then we should just delete the line.

Comment (anmol372) [reply rpm/sign-rpm.sh]: Yes, it's meant t be helpful. Got rid of the echo.

**Review by tbachman — APPROVED** [review]

**Pull request merged**